velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -253,6 +254,24 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, avg) {
   assertPlanConversion(plan, "SELECT avg(c4) FROM tmp");
 }
 
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, subField) {
+  RowTypePtr rowType{ROW(
+      {"c0", "c1", "c2", "c3"}, {BIGINT(), INTEGER(), SMALLINT(), DOUBLE()})};
+  std::vector<RowVectorPtr> vectors;
+  for (int32_t i = 0; i < 10; ++i) {
```

Comment:
No need to create 10 vectors for this test. One vector is sufficient.

Reply:
I fixed this.